qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/AddUserAndAddRolesToUserSteps.java

```diff
@@ -0,0 +1,159 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.Map.Entry;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.page.AddEditUserPage;
+import org.openmrs.contrib.qaframework.page.HomePage;
+import org.openmrs.contrib.qaframework.page.ManageUserPage;
+
+public class AddUserAndAddRolesToUserSteps extends Steps {
```

Comment:
```suggestion
public class UserSteps extends Steps {
```

Reply:
Why change to UserSteps instead of former method.

Reply:
I thought we could make the class name very precise since the implementation shows what the class is performing. I am positive as well with the current name.